### PR TITLE
Try: System font for vanilla editor styles.

### PR DESCRIPTION
**Pull request by jasmussen**
In the older days of the editor, few themes provided editor styles. In order to have contrast between block UI and editor UI, a Serif font was applied. It feels like we can revisit this:

- Using system fonts allows has us load less stuff that is only ever used by themes that don't style the editor.
- Because more themes style the editor, we can reposition the vanilla interface to be more of a writing interface, embrace that it isn't WYSIWYG.
- The serif font hasn't aged well, and a sans-serif appears more legible in the context.

This is not an urgent PR, and I'm sure there will be many opinions. Please do share yours!

Before:

before

After:

after

### CODE REVIEW

File: packages/base-styles/_variables.scss
Comment (jasmussen) [inline packages/base-styles/_variables.scss:16]: This variable has never been used.

File: packages/editor/src/editor-styles.scss
Comment (jasmussen) [inline packages/editor/src/editor-styles.scss:12]: If we were to decide indeed that it was time to change the vanilla font, it would have to happen both here, and in WordPress. So it would need a separate PR, which I will make if this PR gets positive feedback.

**Comment by jasmussen** [comment]
If you would like to test this PR, you have to try it with a theme active which doesn't provide an editor style. The easiest, perhaps, is to search for `add_editor_style( ... ` and comment it out while you test.

File: lib/edit-site-page.php
Comment (aristath) [inline lib/edit-site-page.php:53]: If we switch to system fonts this doesn't need to be translatable since they already work for all locales out of the box. I think we can safely remove the translation function and just add the CSS directly :+1:

**Comment by aristath** [comment]
Overall I'd say this is a great improvement... It solves many issues at the same time.
* Performance issues for those that don't have a fast connection as GFonts is a serious bottleneck in some cases
* Privacy issues with GFonts
* Visual issue. As you mentioned in your OP, the serif font hasn't aged well :laughing:

**Comment by karmatosed** [comment]
I like it as an iteration. It makes sense and also benefits from being more performant. 👍🏻

**Comment by aristath** [comment]
Once this gets merged we can also close this one: https://core.trac.wordpress.org/ticket/46169

**Review by hc0503 — APPROVED** [review]

**Comment by paaljoachim** [comment]
It sounds like a nice win win PR. That will benefit all of us..:)

**Comment by MichaelArestad** [comment]
This works well for me. Nice work!

image

**Comment by jasmussen** [comment]
Thank you Michael! I'll start to pair this up with a core patch, and maybe we can merge it in next week.

**Comment by jasmussen** [comment]
Alright, I tweaked this PR to address the feedback — @aristath did I do it right?

It also now no longer works if you just test the plugin. This has to be paired with https://core.trac.wordpress.org/ticket/46169 to take effect. Would appreciate input on both.

**Review by aristath — APPROVED** [review]
LGTM :+1:

**Comment by garretthyder** [comment]
Just wanted to say thank you all for the work here, looking forward to landing this change into core.

**Comment by jasmussen** [comment]
I made a change as proposed by @aristath to remove the editor font as a dependency. Could use a good sanity check that I did that right before this lands. Thank you.

**Comment by jasmussen** [comment]
I'm having some test failures here that I'm not sure what to do with. Any input would be appreciated.

**Comment by aristath** [comment]
The failing test doesn't seem to be related to the code in this PR... I merged the latest commits from master to this branch, let's see if that fixes the failing test :+1:

**Comment by jasmussen** [comment]
I could use some advice on whether to merge this one in. I know it's dependant on the trac counterpart, so I'd appreciate input on the timing.

**Comment by aristath** [comment]
Hmmm WP5.6 comes out in a few days. In the meantime, WP-trunk is already for 5.7. With that in mind, I think the change in WP can be merged anytime (like... now). If the PR in Gutenberg gets merged now, then worst case scenario: Until WP5.7 comes out, users of the plugin that use a theme which does not define a font-family and falls-back to Noto will see system fonts instead. Which... IMO is fine.

**Comment by jasmussen** [comment]
Thanks. Since I don't think it's particularly critical we get this in 5.6, I'll wait and see if the 5.6 trac patch lands. If it does, we'll merge this right after. Otherwise, it'll be a 5.7 thing. Sound right?

**Comment by aristath** [comment]
> Thanks. Since I don't think it's particularly critical we get this in 5.6, I'll wait and see if the 5.6 trac patch lands. If it does, we'll merge this right after. Otherwise, it'll be a 5.7 thing. Sound right?

Yes, sounds good :+1:

**Comment by jasmussen** [comment]
I've refreshed this one. It would be nice to get this one in 5.7.

**Comment by kellychoffman** [comment]
Ship it.

**Pull request merged**
